src/actions/bits/edit.cr

```diff
@@ -0,0 +1,6 @@
+class Bits::Edit < BrowserAction
+  route do
+    bit = BitQuery.find(params.get(:bit_id))
```

Comment:
You can use “bit_id” so you get type safe params  :)

Reply:
Ah thanks! I forgot you added plain methods to access params.